src/Route/FastPsr7RouteInfo.php

```diff
@@ -1,7 +1,6 @@
 <?php
 namespace Penny\Route;
 
-use InvalidArgumentException;
 
```

Comment:
Thanks @samsonasik  I still have to find a good vim plugin to highlight unused imports :D maybe just running cs fixer can hel me tough